internal/worker/runtime.go

```
@@ -0,0 +1,710 @@
+func (r *DockerRuntime) inspectContainer(ctx context.Context, name string) (Inspection, error) {
+	result, err := r.runDocker(ctx, []string{
+		"container", "inspect",
+		"--format", "{{.State.Running}}\t{{.Config.Image}}",
+		name,
+	})
+	if err != nil {
+		return Inspection{}, err
+	}
+	fields := strings.SplitN(strings.TrimSpace(result.Stdout), "\t", 2)
+	if len(fields) != 2 {
+		return Inspection{}, fmt.Errorf("Docker inspect returned malformed worker state %q", result.Stdout)
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)